src/ParallelAlgorithms/Interpolation/InterpolationTargetDetail.hpp

```diff
@@ -470,6 +475,43 @@ void add_received_variables(
       box);
 }
 
+template <typename SimpleAction, typename Metavariables, typename ArrayIndex,
+          typename Component, typename TemporalId, typename... Args>
+bool functions_of_time_are_ready(Parallel::GlobalCache<Metavariables>& cache,
```

Comment:
I'm going to factor this out into `FunctionsOfTimeAreReady.hpp` in a separate PR because I'll need this elsewhere.